src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -5337,9 +5367,19 @@ def backup_restore(
                 file_name += ".gz"
             files_paths = list(dir_path.glob(f"{file_name}*"))
             if len(files_paths) != 1:
-                raise RuntimeError(
-                    f"{len(files_paths)} files matching the name {file_name} were found in {dir_path}"
-                )
+                if name != "batches":
+                    # All collections except the "batches" one should have exactly one matching file
+                    raise RuntimeError(
+                        f"{len(files_paths)} files matching the name {file_name} were found in {dir_path}"
+                    )
+                if name == "batches" and len(files_paths) > 1:
```

Comment:
Also here it seems that can be reached only if name=="batches"

Reply:
Done